episodes/operations.md

```diff
@@ -51,6 +51,16 @@ in the Incubator is ready to be incorporated into one of The Carpentries _Lesson
 the collection of lessons and curricula that can be taught in an official workshop for
 Software, Library, or Data Carpentry.
 
+|                                       | Incubator          | Lab                |
+| ------------------------------------- | ------------------ | ------------------ |
+| lesson owned by community             | :heavy_check_mark: | :heavy_check_mark: |
+| lesson under development              | :heavy_check_mark: | :x:                |
+| lesson stable                         | :x:                | :heavy_check_mark: |
```

Comment:
We do not force lessons out of the Incubator when they become stable, which is why the table in #183 had a check mark in both columns here. Nevertheless, I think the table is less confusing like this and we should not sacrifice clarity for the sake of accuracy so I am going to merge this as-is.